contracts/modules/assessment/IndividualClaims.sol

```diff
@@ -230,12 +230,31 @@ contract IndividualClaims is IIndividualClaims, MasterAwareV2 {
     uint16 segmentId,
     uint96 requestedAmount,
     string calldata ipfsMetadata
-  ) external payable override onlyMember whenNotPaused returns (Claim memory) {
-    require(
+  ) external payable override onlyMember whenNotPaused returns (Claim memory claim) {
+    claim = _submitClaim(coverId, segmentId, requestedAmount, ipfsMetadata, msg.sender);
+    return claim;
+  }
+  function submitClaimOf(
+    uint32 coverId,
+    uint16 segmentId,
+    uint96 requestedAmount,
+    string calldata ipfsMetadata,
+    address owner
+  ) external payable override onlyInternal whenNotPaused {
+    _submitClaim(coverId, segmentId, requestedAmount, ipfsMetadata, owner);
+  }
+
+  function _submitClaim(
+    uint32 coverId,
+    uint16 segmentId,
+    uint96 requestedAmount,
+    string calldata ipfsMetadata,
+    address owner
+  ) internal whenNotPaused returns (Claim memory) {
+  require(
       coverNFT.isApprovedOrOwner(msg.sender, coverId),
-      "Only the owner or approved addresses can submit a claim"
+      "The owner or approved address can submit a claim"
```

Comment:
i think the old text of the revert was fine. it can be either an owner or an approved address, so there's multiple addresses

Reply:
I got a warning that the contract exceeded the limit, so I had to change few  stuff so it would pass the check...

Reply:
are you running with `export ENABLE_OPTIMIZER=1 ; npm run compile` ?
 
 not expecting this to occur with IC.